eoscostarica/lifebank
# fix(backend): check secret in the backend

Resolves #590 

### What does this PR do?
Check secret in the backend side to valid password is right

### How should this be tested?
- Copy graphql login query to postman
- Make query call
- Check backend don't return any token
